app/views/prison/visits/process_visit.html.erb

```diff
@@ -11,6 +11,7 @@
     <%= hidden_field_tag 'visit[visitor_ids][]', visitor.id  %>
   <% end %>
   <%= f.hidden_field :prison_id  %>
+  <%# f.hidden_field_tag 'prisoner[nomis_offender_id]', @visit.nomis_offender_id %>
```

Comment:
@alan mentioned we needed the offender_id as a hidden field, but left this commented out.